sycl/test-e2e/lit.cfg.py

```diff
@@ -159,6 +159,7 @@
     config.available_features.add(
         "matrix-tf32"
     )  # PVC implies the support of TF32 matrix
+    config.available_features.add("gpu-intel-pvc")
```

Comment:
I'd guess it's not here on purpose - we should use auto-detect for PVC.

Reply:
Fixed in https://github.com/intel/llvm/pull/14720/commits/f6f64954607cb0730d365b2a063ceeadb20ff0f9